pkg/scalers/azure_pipelines_scaler.go

```diff
@@ -139,7 +139,7 @@ func (s *azurePipelinesScaler) GetAzurePipelinesQueueLength(ctx context.Context)
 	r.Body.Close()
 
 	if !(r.StatusCode >= 200 && r.StatusCode <= 299) {
-		return -1, fmt.Errorf("azure Devops REST api returned error. status: %d response: %s", r.StatusCode, string(b))
+		return -1, fmt.Errorf("Azure DevOps REST api returned error. url: %s status: %d response: %s", string(url, r.StatusCode, string(b))
```

Comment:
I think that you have typo here.
There is missing `)` at the end
`return -1, fmt.Errorf("Azure DevOps REST api returned error. url: %s status: %d response: %s", string(url, r.StatusCode, string(b)))`

Reply:
Sorry